x/upgrade/keeper/keeper.go

```diff
@@ -491,7 +491,8 @@ func (k Keeper) DumpUpgradeInfoToDisk(height int64, p types.Plan) error {
 // GetUpgradeInfoPath returns the upgrade info file path
 func (k Keeper) GetUpgradeInfoPath() (string, error) {
 	upgradeInfoFileDir := path.Join(k.getHomeDir(), "data")
-	if err := os.MkdirAll(upgradeInfoFileDir, os.ModePerm); err != nil {
```

Comment:
@julienrbrt will anyone other than the sdk need to modify these files?

Reply:
Mmh, cosmovisor needs to be able to read it and overwrite it if needed.